src/lib.rs

```diff
@@ -2092,6 +2092,21 @@ impl Build {
                         cmd.push_cc_arg("-pthread".into());
                     }
                 }
+
+                if target.os == "nto" {
+                    let arg = match target.arch {
```

Comment:
Could we link to the docs for this? Something like:

```suggestion
                    // Select the target with `-V`. Only supported by qcc:
                    // https://www.qnx.com/developers/docs/7.0.0/index.html#com.qnx.doc.neutrino.utilities/topic/q/qcc.html
                    let arg = match target.arch {
```

Reply:
Done